setup/README.md

```diff
@@ -7,7 +7,7 @@ A guide to setting up your Raspberry Pi
 ### Essential (for general use)
 
 - [SD Card](../installation/sd-cards.md)
-    - We recommend an 8GB class 4 SD card, ideally preinstalled with [NOOBS](../installation/noobs.md).
+    - We recommend a minimum of 8GB class 4, or class 10 SD card. This can be preinstalled with [NOOBS](../installation/noobs.md) or [Raspbian](../installation/installing-images/README.md) to save time, although setting up your own card is easy.
 - [Display and connectivity cable](monitor-connection.md)
     - Any HDMI/DVI monitor and any TV should work as a display for the Pi. For best results, use one with HDMI input, but other connections are available for older devices.
```

Comment:
Would "Any HDMI/DVI monitor and any TV" be better as "Any HDMI/DVI monitor or any TV" ?